hi-ml-cpath/src/health_cpath/utils/viz_utils.py

```diff
@@ -270,9 +272,30 @@ def plot_normalized_confusion_matrix(cm: np.ndarray, class_names: Sequence[str])
     return fig
 
 
+def plot_normalized_and_non_normalized_confusion_matrices(
+    cm: np.ndarray, cm_n: np.ndarray, class_names: Sequence[str],
+) -> plt.Figure:
+    """Plots a normalized and non-normalized confusion matrix and returns the figure.
+
+    :param cm: Non normalized confusion matrix to be plotted.
+    :param cm_n: Normalized confusion matrix to be plotted.
+    :param class_names: List of class names.
+    """
+    fig, axes = plt.subplots(1, 2, figsize=(15, 5))
+    axes[0] = sns.heatmap(cm, annot=True, cmap="Blues", fmt="d", ax=axes[0])
```

Comment:
what are your thoughts of having the two values in one matrix, instead of two different images? E.g. 20% (10/50)

Reply:
as discussed offline, we can address it in a separate PR